src/packages/BalancedJs/contracts/Governance.ts

```diff
@@ -60,9 +60,13 @@ export default class Governance extends Contract {
     return this.call(callParams);
   }
 
-  getProposals(offset: number, batch_size: number = 20) {
+  getProposals(offset: number, batch_size: number) {
```

Comment:
why changed?

Reply:
@octocat512 (@tuan-tl ) Because of submitting many proposals, it doesn't show from item 21 onwards